### PR TITLE
Audit /docs/articles

### LINKED ISSUE #802: Audit All Articles: docs/articles
Review contents of `docs/articles/`

- Reduce passive and conversational tones
- Review images
 - Remove all **unnecessary** screenshots
 - Replace outdated if able

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #802 

Reduce passive voice/conversational tones 
Simplify instructions throughout directory

## Redirect Needed
**Moved:** /docs/articles/updating-payment-methods.md → /docs/articles/sites/updating-payment-methods.md

### CODE REVIEW

File: source/docs/articles/frequently-asked-questions.md [hunk 1]
```diff
@@ -45,13 +45,13 @@ Direct SSH access is not supported, but you are able to directly interface with
...
-
-In order for your site to begin "listening" for your domain, you must first become a paying customer. We consider placing a real domain on on a site to be the point at which the site starts to go live.
-
+Pantheon can handle any domain name you point at it, however DNS configuration is still your responsibility. For more information, see [Domains and SSL](/docs/articles/sites/domains/) and [Going Live](/docs/articles/going-live/).
 
 ## Developing Sites
 
+#### Does Pantheon offer professional services?
+Pantheon provides an infrastructure for professional web developers at agencies and development shops everywhere. We do not build sites or offer professional services. There is a rich ecosystem of Pantheon partners who provide Drupal and WordPress services. See the [Pantheon Partner Directory](https://www.pantheon.io/partners) for more information.
```
Comment (bmackinney) [inline source/docs/articles/frequently-asked-questions.md]: Start with a "No."?

File: source/docs/articles/local.md [hunk 2]
```diff
@@ -5,13 +5,11 @@ category:
...
+While Pantheon provides several options for on-server development, local development has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions, however we can provide some suggestions and known working solutions.  
 If you're looking for a self-contained local development solution on Mac, check out [Kalabox](http://www.kalamuna.com/products/kalabox), which integrates with the Pantheon platform.
 
-## Set up Local Development for a Pantheon Site
-
-To begin, you'll need:
+## Before You Begin
+Be sure you have:
 
 - A local development stack. Pantheon uses a [particular architecture to maximize performance and availability](/docs/articles/sites/all-about-application-containers/), but it's possible to run the same code on a different configurations. As long as the solution supports a minimum of PHP 5.3 and MySQL, you should be fine.  [MAMP](http://www.mamp.info/),  [WAMP](http://www.wampserver.com/) and  [XAMPP](http://www.apachefriends.org/en/xampp.html) all work.
```
Comment (bmackinney) [inline source/docs/articles/local.md]: Add link to kalabox for the adventurous

File: source/docs/articles/organizations.md [hunk 3]
```diff
@@ -5,22 +5,27 @@ category:
...
 
-## Multidev
+- From the User Dashboard, click the **Organizations** tab.
+- From the Site Dashboard, click the Organization's name next to your site's service level.
+- From any Dashboard location, hit the "s" keyboard shortcut to toggle a list of sites and organizations.
 
-All Partner organizations have Multidev available on every site. Click on **Multidev** in your Site Dashboard to get going. Sites have a maximum of ten Multidev environments and unlimited git branches, from which Multidev environments can be created on-demand. Learn how to [use Multidev](/source/docs/articles/sites/multidev).
+
+## Multidev
+All sites associated with any Partner organization will have [Multidev](/docs/articles/sites/multidev) enabled with up to 10 environments at their disposal.
```
Comment (bmackinney) [inline source/docs/articles/organizations.md]: add caveat about multidev envs only being accessible for organization members, not site team member from outside org

**Comment by rachelwhitton** [comment]
@bmackinney - updated with your suggestions. Thanks for the review! Can I get a +1?

**Pull request merged**